lib/lintSource.js

```diff
@@ -88,7 +87,7 @@ function lintPostcssResult(
   postcssResult.stylelint.quiet = config.quiet
 
   const newlineMatch = postcssResult.root.toResult().css.match(/\r?\n/)
-  const newline = newlineMatch ? newlineMatch[0] : os.EOL
+  const newline = newlineMatch ? newlineMatch[0] : "\n"
```

Comment:
I don't think I understand the problem here — I think I need more explanation. `newlineMatch` is supposed to find the first newline that's used in the file, so we can use that newline style again. If no newline is found, we fall back to `os.EOL`. Why would we only want to fall back to `\n`? Can you give an example of the bug that this fixes?